### PR TITLE
[VSTS] Perform configuration pkgs in the pipeline

### PR DESCRIPTION
**Pull request by mandel-macaque**
Get the vars, parse them, ensure we set a output var for the pipeline. We are not yet deciding what steps to run atm.

### CODE REVIEW

File: tools/devops/provision-brew-packages.csx [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+    "cmake", 
     "autoconf",
     "automake",
     "libtool",
     "p7zip",
     "python",
     "libmagic", 
-    "msitools",
-    "wget");
+    "msitools"
```
Comment (spouliot) [inline tools/devops/provision-brew-packages.csx:9]: no `,` ?

Comment (mandel-macaque) [reply tools/devops/provision-brew-packages.csx:9]: good catch, it is working because I disabled the step

**Review by mjbond-msft — APPROVED** [review]
👍 Looks good to me

**Review by tj-devel709 — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**